README.md

```diff
@@ -30,6 +30,7 @@ To add your own project below, just [edit](https://github.com/wwdc/2020/edit/mas
 |[Can Balkaya](https://twitter.com/canbalkya)|[Github](https://github.com/canbalkya/Evape-WWDC20)|[YouTube](Later)|SwiftUI|Submitted|
 |[Christian P](https://twitter.com/Priva2804)|[Github](https://github.com/Priva28/PlanetARium)|[Youtube](https://youtu.be/EwPspV8ZUp4)|SwiftUI, ARKit, SceneKit, Vision|Submitted|
 |[Dave Jha](https://www.davejha.me)|[GitHub](https://github.com/DaveJha/Social-Distancing-Simulator)||SwiftUI, SpriteKit|Submitted|
+|[Daniel (Shao Cheng) Pan](https://github.com/Majestic-Hero)|[GitHub](https://github.com/Majestic-Hero/WWDC-2020-Submission)|[Youtube](https://www.youtube.com/watch?v=eyAY9Dkrsak&t=5s)|SpriteKit, UIKit|Submitted|
```

Comment:
Your YouTube link has a time code in it (`&t=5s`). You should probably remove it

Reply:
do you know where i can revise it?

Reply:
You should just be able to edit the file within your branch to remove that bit

Reply:
I can't find my branch anywhere sorry, would it be fine if I just create a new pull request?

Reply:
It’s linked at the top of the PR page. https://github.com/Majestic-Hero/2020/tree/patch-1